Gemfile

```diff
@@ -40,6 +40,7 @@ group :test, :development do
   gem 'simplecov', :require => false
   gem "codeclimate-test-reporter", "~> 1.0.0"
   gem "capybara"
+  gem "launchy"
```

Comment:
i added that so we could use `save_and_open_page` from capybara for debugging. it's not mission critical so I can remove that if you want :)